specification/suppressions.yaml

```diff
@@ -665,6 +684,7 @@
     - options.@azure-tools/typespec-ts.package-details.name
     - options.@azure-tools/typespec-ts.package-dir
     - options.@azure-tools/typespec-ts.emitter-output-dir
+    - options.@azure-tools/typespec-csharp.*
```

Comment:
@JoshLove-msft can you confirm if we want to suppress `csharp` rules for `keyvault` specs?